toucan_connectors/common.py

```diff
@@ -71,26 +71,25 @@ def _render_query(query, parameters):
             return {key: _render_query(value, parameters) for key, value in deepcopy(query).items()}
         elif isinstance(query, list):
             return [_render_query(elt, parameters) for elt in deepcopy(query)]
-        elif type(query) is str:
+        elif isinstance(query, str):
             if not _has_parameters(query):
                 return query
-            clean_p = deepcopy(parameters)
+
+            # Replace param templating with jinja templating:
+            query = re.sub(RE_PARAM, r'{{ \g<1> }}', query)
```

Comment:
So easier ^^

By the way, I think we should progressively retire this parameter syntax in favor of Jinja one, which is simpler to read and write. Also, it's complex for users to have two possibilities that does the same thing.
Creating an issue to suggest that: https://github.com/ToucanToco/toucan-connectors/issues/303